scripts/fix-daily-branch

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Fix daily build branches so daily build recipes by dropping all 'cherry-pick'
+# quilt patches applied to a given ubuntu/* release branch.
+#  Any *cpick* files in debian/patches are git revert'd in reverse order
+# so that the upstream commits the *cpick* file represent are able to apply.
+set -e
+
+error() { echo "$@" 1>&2; }
+fail() { [ $# -eq 0 ] || error "$@"; exit 1; }
+
+gitcmd() {
+    git "$@" || fail "Failed running git $@"
+}
+
+Usage() {
+cat <<EOF
+${0##*/}
+
+Create a daily_branch which reverts any debian/patches/*cpick* commits from
+source_branch.
+
+options:
+   -h|--help                   show this message.
+   --source-branch          The local branch name from which to revert all
+                            debian/patches/*cpick* files. Default: ubuntu/devel.
+   --daily-branch           The local daily branch name to create which reverts
+                            debian/patches/*cpick* files. Default:
+                            ubuntu/daily/devel
+EOF
+}
+
+drop_cherry_picks_from_daily_branch() {
+    local source_branch="${1}" daily_branch="${2}" daily_remote="${3:-origin}"
```

Comment:
We only need to do this defaulting in one place:

```suggestion
    local source_branch="${1}" daily_branch="${2}" daily_remote="${3}"
```

Reply:
dropped.